package.json

```diff
@@ -24,9 +24,10 @@
   "private": false,
   "scripts": {
     "dev": "nodemon .",
+    "dev:api": "nodemon src/api/index.js",
     "lint": "eslint .",
     "lint:fix": "eslint --fix .",
-    "test": "echo No tests"
+    "test": "jest ."
```

Comment:
isn't just `jest` enough?

also, let's make this a `watchAll` and create another `test:ci` which doesnt have it (and update `.circleci/config`)